observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dashboard-ui-plugin"]
+= Logging UI plugin
+include::_attributes/common-attributes.adoc[]
+:context: logging-ui-plugin
+
+toc::[]
+
+:FeatureName: The {coo-full}
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+
+The logging UI plugin surfaces logging data in the {product-title} web console on the *Observe* -> *Logs* page. 
+You can specify filters, queries, time ranges and refresh rates, with the results displayed as a list of collapsed logs, which can then be expanded to show more detailed information for each log.
+
+When you have also deployed the Troubleshooting UI plugin, it connects to the Korrel8r service and adds direct links from the Administration perspective, from the **Observe** -> **Logs** page, to the **Observe** -> **Metrics** page with a correlated PromQL query. It also adds a **See Related Logs** link from the Administration perspective alerting detail page, at **Observe** -> **Alerting**,  to the **Observe** -> **Logs** page with a correlated filter set selected.
```

Comment:
the OCP must be 4.16+ for troublesboot plugin

Reply:
Done - and added to overview, and troubleshooting sections